pasta/base/ast_utils.py

```diff
@@ -182,3 +184,16 @@ def get_last_child(node):
     if node.handlers:
       return get_last_child(node.handlers[-1])
   return node.body[-1]
+
+def remove_child(parent, child):
+  parent_list = None
+  for _, field_value in ast.iter_fields(parent):
+    if isinstance(field_value, list) and child in field_value:
+      parent_list = field_value
+      break
+  else:
+    raise errors.InvalidAstError('Unable to find list containing child %r on '
+                                 'parent node %r' % (child, parent))
+
+  parent_list.remove(child)
```

Comment:
while I'm nitpicking, you could just remove it when you find it (192) instead of storing this.

Reply:
Done